server/chat-plugins/room-events.ts

```diff
@@ -19,30 +34,47 @@ function formatEvent(event: RoomEvent, showAliases?: boolean) {
 	if (!isNaN(timeRemaining)) {
 		explanation = `This event will start in: ${Chat.toDurationString(timeRemaining, {precision: 2})}`;
 	}
+
+	const eventID = toID(event.eventName);
+	const aliases = getAllAliases(room).filter(
+		alias => (room.settings.events?.[alias] as RoomEventAlias).eventID === eventID
+	);
+	const categories = getAllCategories(room).filter(
+		category => (room.settings.events?.[category] as RoomEventCategory).events.includes(eventID)
+	);
+
 	let ret = `<tr title="${explanation}">`;
 	ret += Utils.html`<td>${event.eventName}</td>`;
-	ret += showAliases ? Utils.html`<td>${event.aliases?.join(", ")}</td>` : ``;
+	if (showAliases) ret += Utils.html`<td>${aliases.join(", ")}</td>`;
+	if (showCategories) ret += Utils.html`<td>${categories.join(", ")}</td>`;
 	ret += `<td>${Chat.formatText(event.desc, true)}</td>`;
 	ret += Utils.html`<td><time>${event.date}</time></td></tr>`;
 	return ret;
 }
 
 function getAllAliases(room: Room) {
-	if (!room.settings.events) return [];
```

Comment:
?

This should be restored; `room.settings.events` should not be taken as guaranteed to exist.